src/app/ubs-admin/components/ubs-admin-orders/ubs-admin-orders.component.ts

```diff
@@ -0,0 +1,32 @@
+import { AdminOrdersService } from './../../services/admin-orders.service';
+import { Component, OnInit } from '@angular/core';
+import { takeUntil } from 'rxjs/operators';
+import { Subject } from 'rxjs';
+
+@Component({
+  selector: 'app-ubs-admin-orders',
+  templateUrl: './ubs-admin-orders.component.html',
+  styleUrls: ['./ubs-admin-orders.component.scss']
+})
+export class UbsAdminOrdersComponent implements OnInit {
+  destroy: Subject<boolean> = new Subject<boolean>();
+  orders = [];
+  isAnyOrders: boolean;
+
+  constructor(private adminOrdersService: AdminOrdersService) {}
+
+  ngOnInit() {
+    this.adminOrdersService
+      .getOrders()
+      .pipe(takeUntil(this.destroy))
+      .subscribe((item) => {
+        this.orders = item;
+      });
+  }
+
+  changeCard(id) {
+    this.orders.map((el) => {
+      el.id === id && el.extend ? (el.extend = false) : el.id === id ? (el.extend = true) : (el.extend = el.extend);
```

Comment:
We should change it to if-else statement
Hard to read :/

Reply:
As for me this is better... 
Also we will change this logic when we will recive real data.